### PR TITLE
#701 scheduler web dialog updates

### PR DESCRIPTION
**Pull request by proddy**

### CODE REVIEW

**Comment by proddy** [comment]
still need to look at the backend C code to make sure id's are generated in sequence and unique

File: src/web/WebSchedulerService.cpp [hunk 1]
```diff
@@ -37,14 +37,14 @@ void WebSchedulerService::begin() {
...
-        JsonObject si     = schedule.createNestedObject();
-        si["id"]          = scheduleItem.id; // name, is unqiue
-        si["active"]      = scheduleItem.active;
-        si["flags"]       = scheduleItem.flags;
-        si["time"]        = scheduleItem.time;
-        si["cmd"]         = scheduleItem.cmd;
-        si["value"]       = scheduleItem.value;
-        si["description"] = scheduleItem.description;
+        JsonObject si = schedule.createNestedObject();
+        si["id"]      = scheduleItem.id; // name, is unqiue
```
Comment (MichaelDvP) [inline src/web/WebSchedulerService.cpp:41]: The id is undefined here, go back to `smallitoa(counter)`, or skip id here and generate all unique ids in webpage on load.

Comment (proddy) [reply src/web/WebSchedulerService.cpp:41]: yes, that's next to do. see the previous comment

Comment (proddy) [reply src/web/WebSchedulerService.cpp]: fixed.

**Review by MichaelDvP — APPROVED** [review]
There are a lot of small changes i don't understand in all consequences, but all seem to work well.
I liked the checkicon more than the circle, but it's also ok.

**Comment by proddy** [comment]
> There are a lot of small changes i don't understand in all consequences, but all seem to work well. I liked the checkicon more than the circle, but it's also ok.

yeah i know what you mean. There's always a debate on how to show active/no-active states. Like on https://ux.stackexchange.com/questions/100504/how-should-i-show-disabled-state-in-table

The problem with a CheckIcon/Tick the user will think it's actually selectable, when it isn't. 

I'll make a chage to use round colored circles and ticks

**Pull request merged**